### PR TITLE
Design review remediations: guides

### PR DESCRIPTION
**Pull request by lauraGgit**
Change scss to use variable functions so that uswds theme settings show up.

Guides page update review #3367

Section header will be in another PR

## Guide Links

- [x] All links are #1C304A and #046B99 on hover
- [x] Caret icon is vertically centered with the text
- [x] Caret icon is ~ the height of the letter "x" in the the text 

CTA

- [x]  Background color should be #1C304A
- [x]  Text is lg size
- [x]  Button has background color #00CFFF and font-color #1C304A. (can follow the ACCENT COOL COLOR example for states)

FOOTER REVIEW #3368 
- [x] Headers are size `lg`
- [x] Headers are static text with no interaction effects. 
- [x] Links are `#046B99`

### CODE REVIEW

File: _sass/_uswds-theme-custom-styles.scss
Comment (igorkorenfeld) [reply _sass/_uswds-theme-custom-styles.scss:48]: @lauraGgit flagging this one since I think you said you care about the indents

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
Looking good, only see two things
1. The space after the intro hero should be 5 units, which will be addressed via a global change
2. The intro text (the first paragraph below the intro-hero), we should make `lg` on all pages now.

File: _sass/_uswds-theme-custom-styles.scss
Comment (lauraGgit) [reply _sass/_uswds-theme-custom-styles.scss]: good flag!

**Comment by lauraGgit** [comment]
ok @igorkorenfeld pushed up a couple of hound fixes and made the intro text large.

#3386 has the global change so if you don't mind me holding off then i won't have to undo some conflicts

**Review by igorkorenfeld — APPROVED** [review]
That sounds good, I'll look double check it once we merge that one in then. In that case, everything looks good to me!

I'll let you merge this in because you have a better sense of the branches and where this goes than I do.

**Pull request merged**
